saleor/plugins/webhook/plugin.py

```diff
@@ -2483,6 +2483,62 @@ def translation_updated(
             )
         return previous_value
 
+    def translations_created(
```

Comment:
What about the previously used methods? `translation_created` and `translation_updated`?

Reply:
We should also change `BasePlugin` definition with replacement of old methods

Reply:
I will remove usage of the old methods and add definition to `BasePlugin` as it is missing currently.